docs/source/operations/corda_networkyaml.md

```diff
@@ -19,13 +22,17 @@ The configurations are grouped in the following sections for better understandin
 
 * organizations
 
-`type` defines the platform choice like corda/fabric.
+Here is the snap shot of the same from the sample network.yaml
```

Comment:
Please remove the phrase "the same" from this whole document and rephrase. also snapshot is a single word